### PR TITLE
fixed scrolling for connection viewlet

### PR DESCRIPTION
**Pull request by abist**
Fixes https://github.com/Microsoft/azuredatastudio/issues/4048

### CODE REVIEW

File: src/sql/parts/dataExplorer/objectExplorer/connectionViewlet/connectionViewletPanel.ts [hunk 1]
```diff
@@ -111,7 +111,7 @@ export class ConnectionViewletPanel extends ViewletPanel {
 
 	layoutBody(size: number): void {
 		this._searchBox.layout();
-		this._serverTreeView.layout(size); // account for search box
+		this._serverTreeView.layout(size - 46); // account for search box and horizontal scroll bar
```
Comment (kburtram) [inline src/sql/parts/dataExplorer/objectExplorer/connectionViewlet/connectionViewletPanel.ts]: is this going to do the right thing when the horizontal scrollbar isn't visible?

**Comment by anthonydresser** [comment]
I think we need some PM/Design @yualan input here. Typically everywhere else we have tree in viewlets (every viewlet) the trees don't have a horizontal scroll, they just have ellipse if the content goes over.

File: src/sql/parts/dataExplorer/objectExplorer/connectionViewlet/connectionViewletPanel.ts (hunk 1 above)
Comment (abist) [reply src/sql/parts/dataExplorer/objectExplorer/connectionViewlet/connectionViewletPanel.ts]: Yes, the searchBox is 36px and the scrollBar is 10px.

**Comment by kburtram** [comment]
@anthonydresser @abist can we at least fix the regression with the vertical scrollbar discussed in https://github.com/Microsoft/azuredatastudio/issues/4048?  You can work on what to do about horizontal scrolling separately if desired.

**Comment by abist** [comment]
@yualan We can perhaps open a new issue to track "horizontal scrolling"

**Comment by yualan** [comment]
https://github.com/Microsoft/azuredatastudio/issues/4106 done

**Pull request merged**